tests/test_wave_01.py

```diff
@@ -130,14 +140,19 @@ def test_update_task_not_found(client):
 
     # Assert
     assert response.status_code == 404
+    assert response_body == {"message": "Task 1 not found"}
+    assert Task.query.all() == []
```

Comment:
In this case, this `assert` does help verify a use path, namely that the call to `PUT /tasks/1` didn't somehow add an extra task. So definitely think about how the exact same `assert` has different testing power in different cases. :3